api/python/quilt3/packages.py

```diff
@@ -63,6 +65,44 @@ def _delete_local_physical_key(pk):
     pathlib.Path(parse_file_url(urlparse(pk))).unlink()
 
 
+class ObjectCache(object):
```

Comment:
this class should have a `CACHE_SCHEMA_VERSION` (and we should check it in the `.get` and `.set` routines?) so that if we change cache formats we can force clients to upgrade.

what happens if we decide to change format or directory location. do we want the cache to be namespaced under a `CACHE_SCHEMA_VERSION` folder so that different client versions can run simultaneously?

Reply:
Yeah, I'll just add a `v0` to the path.